packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -92,6 +93,19 @@ export const SubscriptionCreate = ({
 
   const [paypalScriptLoaded, setPaypalScriptLoaded] = useState(false);
 
+  // The Stripe customer isn't created until payment is submitted, so
+  // customer can be null and customer.payment_provider can be undefined.
+  const paymentProvider: ProviderType = customer?.payment_provider;
+
+  const isStorybookContext = Array.from(
+    document.head.getElementsByTagName('SCRIPT')
+  ).filter(
+    (ele) =>
+      // @ts-ignore: HTMLElement does not have property 'src'
+      ele.src ===
+      'https://www.paypal.com/sdk/js?client-id=sb&vault=true&commit=false&intent=authorize&disable-funding=credit,card'
+  ).length;
```

Comment:
I added a `ts-ignore` comment for a storybook-only code block in `SubscriptionCreate` -- how can I cast this result as an `HTMLScriptElement` array?

Also, in general, this is kind of a hacky way to get the `PaypalButton` to load just for Storybook. Open to alternatives if you have them!

Reply:
`getElementsByTagName` takes a type arg so you can write: `    document.head.getElementsByTagName<'script'>('script')
` then everything in that returned collection is a `HTMLScriptElement`.

I don't have an alternative, but, IMHO as they used to say, I'd rather not have the story in storybook than adding code to accommodate it.

Reply:
Hmm... that's a fair point. I thought of a much more minimal approach though where I just declare a global `isStorybookContext` in the Storybook head file and check for that. What do you think?

```diff
diff --git a/packages/fxa-payments-server/.storybook/preview-head.html b/packages/fxa-payments-server/.storybook/preview-head.html
index e0bb2403e..f041f156e 100644
--- a/packages/fxa-payments-server/.storybook/preview-head.html
+++ b/packages/fxa-payments-server/.storybook/preview-head.html
@@ -1,2 +1,5 @@
+<script>
+    const isStorybookContext = true;
+</script>
 <script src="https://js.stripe.com/v3/"></script>
 <script src = "https://www.paypal.com/sdk/js?client-id=sb&vault=true&commit=false&intent=authorize&disable-funding=credit,card"></script>
diff --git a/packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx b/packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx
index 111c6204d..1ddfc1487 100644
--- a/packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx
+++ b/packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx
@@ -38,6 +38,8 @@ const PaypalButton = React.lazy(() => import('../../Product/PayPalButton'));
 type PaymentError = undefined | StripeError;
 type RetryStatus = undefined | { invoiceId: string };
 
+declare var isStorybookContext: boolean | undefined;
+
 export type SubscriptionCreateStripeAPIs = Pick<
   Stripe,
   'createPaymentMethod' | 'confirmCardPayment'
@@ -97,26 +99,12 @@ export const SubscriptionCreate = ({
   // customer can be null and customer.payment_provider can be undefined.
   const paymentProvider: ProviderType = customer?.payment_provider;
 
-  const isStorybookContext = Array.from(
-    document.head.getElementsByTagName('SCRIPT')
-  ).filter(
-    (ele) =>
-      // @ts-ignore: HTMLElement does not have property 'src'
-      ele.src ===
-      'https://www.paypal.com/sdk/js?client-id=sb&vault=true&commit=false&intent=authorize&disable-funding=credit,card'
-  ).length;
-
   useEffect(() => {
     if (!config.featureFlags.usePaypalUIByDefault) {
       return;
     }
 
-    if (paypalButtonBase) {
-      setPaypalScriptLoaded(true);
-      return;
-    }
-
-    if (isStorybookContext) {
+    if (paypalButtonBase || isStorybookContext) {
       setPaypalScriptLoaded(true);
       return;
     }
```

If you still think it's a bad idea, I can just revert the Storybook changes.